arm-compute/2015-06-15/swagger/compute.json

```diff
@@ -2451,26 +2590,27 @@
         "platformUpdateDomainCount": {
           "type": "integer",
           "format": "int32",
-          "description": "Gets or sets Update Domain count."
+          "description": "Update Domain count."
         },
         "platformFaultDomainCount": {
           "type": "integer",
           "format": "int32",
-          "description": "Gets or sets Fault Domain count."
+          "description": "Fault Domain count."
         },
         "virtualMachines": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/SubResource"
           },
-          "description": "Gets or sets a list containing reference to all Virtual Machines  created under this Availability Set."
+          "description": "A list of references to all virtual machines in the availability set."
         },
         "statuses": {
+          "readOnly": true,
```

Comment:
@jhendrixMSFT , here as well we are just correcting the swagger, the service has not changed